storybook/pages/KeycardEnterPukPagePage.qml

```diff
@@ -0,0 +1,48 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+
+import AppLayouts.Onboarding2.pages 1.0
+
+Item {
+    id: root
+
+    readonly property string existingPuk: "111111111111"
+
+    KeycardEnterPukPage {
+        id: page
+        anchors.fill: parent
+        remainingAttempts: 3
+        tryToSetPukFunction: (puk) => {
+                                 console.warn("!!! ATTEMPTED PUK:", puk)
+                                 const valid = puk === root.existingPuk
+                                 if (!valid)
+                                     remainingAttempts--
+                                 return valid
+                             }
+        onKeycardPukEntered: (puk) => {
+                                 console.warn("!!! CORRECT PUK:", puk)
+                                 console.warn("!!! RESETTING FLOW")
+                                 state = "entering"
+                             }
+        onKeycardFactoryResetRequested: {
+            console.warn("onKeycardFactoryResetRequested")
+            console.warn("!!! RESETTING FLOW")
+            state = "entering"
+            remainingAttempts = 3
+        }
+    }
+
+    Label {
+        anchors.bottom: parent.bottom
+        anchors.right: parent.right
+        text: "Hint: %1".arg(root.existingPuk)
+    }
+}
+
+// category: Onboarding
+// status: good
+// https://www.figma.com/design/Lw4nPYQcZOPOwTgETiiIYo/Desktop-Onboarding-Redesign?node-id=1281-45942&node-type=frame&m=dev
+// https://www.figma.com/design/Lw4nPYQcZOPOwTgETiiIYo/Desktop-Onboarding-Redesign?node-id=1281-45950&node-type=frame&m=dev
+// https://www.figma.com/design/Lw4nPYQcZOPOwTgETiiIYo/Desktop-Onboarding-Redesign?node-id=1281-45959&node-type=frame&m=dev
+// https://www.figma.com/design/Lw4nPYQcZOPOwTgETiiIYo/Desktop-Onboarding-Redesign?node-id=1281-45966&node-type=frame&m=dev
+// https://www.figma.com/design/Lw4nPYQcZOPOwTgETiiIYo/Desktop-Onboarding-Redesign?node-id=1281-45996&node-type=frame&m=dev
```

Comment:
those links seem to lead to improper location

Reply:
Ah right, the new design is not in place yet

Reply:
Fixed